practice/react-practice-one/src/components/InputFields/InputField.jsx

```diff
@@ -0,0 +1,15 @@
+/* Import dependencies */
+import PropTypes from 'prop-types';
+
+/* Import CSS */
+import './inputField.css';
+
+const InputField = ({ type }) => {
+  return <input type={type} className="input-container" />;
```

Comment:
How about `TextField`?

Reply:
@JennyDoJD This component should have more props. 
Will you use this component in ProductF from? If yes, pls consider the props need in this component